### PR TITLE
Pr update procedural finger

### PR DESCRIPTION
**Pull request by peyronq1**
Hello everyone,

I iterated on the soft finger design tutorial and I think I'm ready to share it. First of all, I think we should rename the tutorial as "Soft Finger Design", since "Procedural finger" seems a bit abstract to me.  As an overview of the modifications I propose:

Tutorial document

- Organization with steps
- Addition of the last step where the students must prepare the finger 3D printing on CURA
- Iteration on the text with the different crash testers, correction of typos, addition of details concerning the design process
- Update of the images when needed

Mesh generation

- Robustification of the mesh generation code following the crash tester experiences

SOFA scene

- Improvement of the contact force computation
- Choice of a cylinder as the object to grasp, as it will be easier to do physically than a sphere
- Inclusion of the flexible filament mechanical properties (provided by the manufacturer)

The tutorial works on SOFA 22.06.00_beta1 with Python 3.8.10

### CODE REVIEW

File: docs/sofapython3/tutorials/ProceduralFinger/soft-finger-design.md [hunk 1]
```diff
@@ -0,0 +1,284 @@
...
+- The 3D mesh of the finger along with an elastic force field to simulate its deformation
+- A box with rigidified mesh nodes at the left wall base, which pose is updated according to the
+servomotor rotation
+- A box of rigidified mesh nodes at the right wall base which pose is fixed
+- A mesh of the object to grasp, fixed in space.
+- Collision models to account for eventual collisions between the finger and the object, and
+between the inner surfaces of the left and right walls
+
+<pre>
+<a href="details/procedural-finger.py"> <img src="../../../images/icons/play.png" width="14px"/>Try the scene in SOFA.</a>
```
Comment (EulalieCoevoet) [inline docs/sofapython3/tutorials/ProceduralFinger/soft-finger-design.md:182]: If I understand correctly here it should be gripper_simulation.py?

File: docs/sofapython3/tutorials/ProceduralFinger/soft-finger-design.md [hunk 2]
```diff
@@ -0,0 +1,284 @@
...
+<a href="javascript:void(0)" onclick="toggle('step1code');"> <img src="../../../images/icons/play.png" width="14px"/>Show/Hide the code.</a>
+</pre>
+[comment]: <> (<div id='step1code' class='hide'>)
+[comment]: <> (```python)
+[comment]: <> (..autofile::procedural-finger.py)
```
Comment (EulalieCoevoet) [inline docs/sofapython3/tutorials/ProceduralFinger/soft-finger-design.md:187]: same here

**Comment by EulalieCoevoet** [comment]
Otherwise it looks nice!

File: docs/sofapython3/tutorials/ProceduralFinger/soft-finger-design.md (hunk 1 above)
Comment (peyronq1) [reply docs/sofapython3/tutorials/ProceduralFinger/soft-finger-design.md:182]: Ow I didn't see that, yes it should!

File: docs/sofapython3/tutorials/ProceduralFinger/soft-finger-design.md (hunk 2 above)
Comment (peyronq1) [reply docs/sofapython3/tutorials/ProceduralFinger/soft-finger-design.md:187]: I put this part in comments as this was not working well on my Markdown editor (PyCharm). But yes procedural-finger.py should become gripper_simulation.py

**Pull request closed without merging**